toptal/picasso
# fix: revert adding postfix to input

Reverts toptal/picasso#2207

### Description

Reverting this [ticket](https://toptal-core.atlassian.net/browse/FX-2224) because it will cause feature tests to fail because they rely on native input name to locate inputs.

Also add a `console.warn` when an input loses autocomplete feature.

### How to test

1. Go to an example in picasso forms
2. Change one input's name to `input1` or `title`, etc
3. A warning should appear in console and input name stays the same

### Screenshots

![image](https://user-images.githubusercontent.com/18625278/140856570-b660a302-356e-42c1-9a31-028fbcca0131.png)

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
